### PR TITLE
Repackaged for updated in Analytical Rule

### LINKED ISSUE #11215: Suspicious process creation analytics rule logic error
**The problem**
The "Process Creation with Suspicious CommandLine Arguments" analytics rule contains a type error on the 3rd line of the query where an array is passed to the todynamic() function, which only accepts strings. See first screenshot.

Additionally, the operations on the 4th and 5th lines of the query expect the same variable to be both a string and dynamic type. Please look at the 2nd screenshot for clarity.

**To Reproduce**
1. Open the rule in the analytics rule wizard
2. Go to set rule logic
3. Scroll down to the 3rd line of the query

**Screenshots**
1
image
2
image

### PR DESCRIPTION
**Pull request by v-rusraut**
Required items, please complete
   
   Change(s):
   - Repackaged

   Reason for Change(s):
  - Fixed Issue - https://github.com/Azure/Azure-Sentinel/issues/11215
   - Updated Analytical Rule - Process Creation with Suspicious CommandLine Arguments 

   Version Updated:
   - 3.0.1
   Testing Completed:
   - Done

### CODE REVIEW

File: Solutions/Malware Protection Essentials/Analytic Rules/SuspiciousProcessCreation.yaml [hunk 1]
```diff
@@ -42,26 +42,23 @@ relevantTechniques:
...
+  DvcDomain,
+  TargetUsername,
+  TargetUsernameType,
+  TargetProcessName,
+  TargetProcessId,
+  CommandLine
+  | extend Username = iff(tostring(TargetUsernameType) == 'Windows', tostring(split(TargetUsername, '\\')), TargetUsername)
+  | extend NTDomain = iff(tostring(TargetUsernameType) == 'Windows', tostring(split(TargetUsername, '\\')), TargetUsername)
+  | extend Username = iff(tostring(TargetUsernameType) == 'UPN', tostring(split(TargetUsername, '@')), Username)
+  | extend UPNSuffix = iff(tostring(TargetUsernameType) == 'UPN', tostring(split(TargetUsername, '@')), '')
```
Comment (rahul0216) [inline Solutions/Malware Protection Essentials/Analytic Rules/SuspiciousProcessCreation.yaml:61]: Missed the subscripts after split.

**Review by rahul0216 — COMMENTED** [review]
Please update the code for inline query.

File: Solutions/Malware Protection Essentials/Analytic Rules/SuspiciousProcessCreation.yaml (hunk 1 above)
Comment (v-rusraut) [reply Solutions/Malware Protection Essentials/Analytic Rules/SuspiciousProcessCreation.yaml]: done

**Review by rahul0216 — APPROVED** [review]
Changes look good.

**Review by v-atulyadav — APPROVED** [review]

**Pull request merged**